front-end/src/components/MostPopularVoucher.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+    <div id="popularVoucher">
+    <el-card id="chart-container">
+      <fusioncharts
+      :type="type"
+      :width="width"
+      :height="height"
+      :dataformat="dataFormat"
+      :dataSource="dataSource"
+      >
+      </fusioncharts>
+    </el-card>
+  </div>
+</template>
+
+<script>
+import Vue from 'vue';
+import VueFusionCharts from 'vue-fusioncharts';
+import FusionCharts from 'fusioncharts';
+import Charts from 'fusioncharts/fusioncharts.charts';
+import constants from "../constants"
+//import the theme
+import FusionTheme from 'fusioncharts/themes/fusioncharts.theme.fusion'
+
+// register VueFusionCharts component
+Vue.use(VueFusionCharts, FusionCharts, Charts, FusionTheme)
+
+const axios = require('axios').default;
+let api_server = 'http://localhost:3000';
+let scheme = '002'
+// Copy datasource from 'Data' tab
+const dataStore = {}
+
+export default {
+    name: 'popularVoucher',
+    data() {
+        return {
+            type: 'pie2d',
+            width: constants.chart_width,
+            height: constants.chart_height,
+            dataFormat: 'json',
+            renderAt: "chart-container",
+            dataSource: dataStore,
+        }
+    },
+    mounted() {
+        this.getData()
+    },
+    methods: {
+        async getData() {
+            // Get the chart data
+            const formattedData = await axios.get(`${api_server}/most_popular_scheme?scheme=${scheme}`)
+                .then(function (response) {
+                    // Handle success
+                    console.log("Here's the response")
+                    console.log(response.data)
+                         const dataStore = {
+                                "chart": {
+                                "caption": "Most Popular Voucher Size",
+                                paletteColors: constants.palette,
+                                "subCaption": "All time",
+                                "numberPrefix": "Stamps: ",
+                                "showPercentInTooltip": "0",
+                                "decimals": "1",
+                                "useDataPlotColorForLabels": "1",
+                                "theme": "fusion"
+                                },
+                                "data": [
+                                {
```

Comment:
Again, it shouldn't be hard-coded for the number of locations and schemes we were given